packages/fuels/src/cli/utils/fuelsVersionCache.ts

```diff
@@ -0,0 +1,31 @@
+import fs from 'fs';
+import path from 'path';
+
+export const FUELS_VERSION_CACHE_FILE = path.join(__dirname, 'FUELS_VERSION');
+
+export type FuelsVersionCache = string;
+
+export const saveToCache = (cache: FuelsVersionCache) => {
+  fs.writeFileSync(FUELS_VERSION_CACHE_FILE, cache, 'utf-8');
+};
+
+export const FUELS_VERSION_CACHE_TTL = 6 * 60 * 60 * 1000; // 6 hours in milliseconds
+
+export const checkAndLoadCache = (): FuelsVersionCache | null => {
+  const doesVersionCacheExist = fs.existsSync(FUELS_VERSION_CACHE_FILE);
+
+  if (doesVersionCacheExist) {
+    const cachedVersion = fs.readFileSync(FUELS_VERSION_CACHE_FILE, 'utf-8').trim();
+
+    if (!cachedVersion) {
+      return null;
+    }
```

Comment:
Is it possible for the cache file to exist but `cachedVersion` to be null?

Reply:
Yes, in a rare edge case the file can be empty. Not expected but just accounting for all cases here